lib/global_config.js

```diff
@@ -157,3 +158,44 @@ const rest = {
   HTTPS_PROTOCOL: 'https'
 };
 exports.rest = rest;
+
+const defaultXmlParserConfiguration = {
+  ignoreAttributes: true,
```

Comment:
let's add a comment why do we keep those defaults

Reply:
ok